### PR TITLE
Tweaks for the OwO Cretins

### PR DESCRIPTION
**Pull request by DarkoPendragon**

### CODE REVIEW

**Comment by IllusionMan1212** [comment]
Really nice job you did here (better than me tbh), i'll take a better look at the code tomorrow (when i'm not about to fall on my face)

**Comment by DarkoPendragon** [comment]
Thanks, the only problem should be the last commit. My Git program pulled before it made the pr and I didn't realize. Other than that everything should be fine unless I missed something while testing it out.

**Review by IllusionMan1212 — COMMENTED** [review]
Everything looks fine to me, i just have one question, why use 2 space indentation lol. 
tbh the indentation is fine as long as it's consistent.

Also if you could please rebase on latest master and squash your last commits (if you know how, otherwise it's fine) so i can merge.
and thank you!!

File: bot.js [hunk 1]
```diff
@@ -17,27 +21,31 @@ dbl.on('error', err => {
  console.log(`${err}`);
 })
 
-fs.readdir("./events/", (err, files) => {
-  if (err) return console.error(err);
-  files.forEach(file => {
-    const event = require(`./events/${file}`);
-    let eventName = file.split(".")[0];
-    client.on(eventName, event.bind(null, client));
+const start = async () => {
```
Comment (LiamTownsley) [inline bot.js:24]: Isn't that a straight copy+paste from evee?

Comment (canarado) [reply bot.js:24]: evee aint the first bot to use that nor is it the only

Comment (LiamTownsley) [reply bot.js:24]: This is a really old review, accidentally submitted it.

Comment (canarado) [reply bot.js:24]: no problem haha, cheers

**Pull request closed without merging**